unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotsView.cs

```diff
@@ -42,19 +42,30 @@ public void AddSlotToSection(string sectionName, string slotCategory)
             avatarSlots.Add(slotCategory, avatarSlot);
             avatarSlot.OnSelectAvatarSlot += (slotModel, isToggled) => OnToggleAvatarSlot?.Invoke(slotModel.category, slotModel.allowsColorChange, isToggled);
             avatarSlot.OnUnEquip += (wearableId) => OnUnequipFromSlot?.Invoke(wearableId);
+            avatarSlot.OnFocusHiddenBy += (hiddenBy) => avatarSlots[hiddenBy].ShakeAnimation();
         }
 
-        public void SetSlotNftImage(string category, string imageUrl) =>
-            avatarSlots[category].SetNftImage(imageUrl);
-
-        public void SetSlotRarity(string category, string rarity) =>
-            avatarSlots[category].SetRarity(rarity);
-
         public void DisablePreviousSlot(string category) =>
             avatarSlots[category].OnPointerClickOnDifferentSlot();
 
-        public void ResetCategorySlot(string category) =>
+        public void SetSlotContent(string category, WearableItem wearableItem, string bodyShape)
```

Comment:
Not really because the slot model has some things coming from different places, like the "allowColorChange", set on initialization and never changed, not depending on the wearable definition